src/SDK.php

```diff
@@ -82,6 +82,26 @@ public function authenticate(\Moltin\SDK\AuthenticateInterface $auth, $args = ar
         return ($this->token === null ? false : true);
     }
 
+    public function switchStore($store)
+    {
+        $result = \Moltin::Get('account/stores/switch/'.$_POST['store']);
```

Comment:
We could do that I guess, so are you suggesting to remove the switch store end point?